dap-mode.el

```diff
@@ -758,6 +788,12 @@ thread exection but the server will log message."
                                 (message "Unable to find handler for %s." (pp parsed-msg)))))))
             (dap--parser-read parser msg)))))
 
+(defun dap--create-output-buffer (session-name)
+  "Creates an output buffer with with name SESSION-NAME."
+  (with-current-buffer (get-buffer-create (concat "*" session-name " out*"))
+    (set (make-local-variable 'window-point-insertion-type) t)
```

Comment:
having this "hardcoded" will still keep the other issue(stealing the focus). To solve it, I propose adding a minor mode (e. g. `dap-output-buffer-mode` and enabling it in the output buffer. Then, if the user wants to follow the output he/she could write 

```
(add-hook 'dap-output-buffer-mode-hook (lamnda () (make-local-variable 'window-point-insertion-type))
```

Reply:
... FTR we need that minor mode either way, since so the users could have custom keybindings in the output and so on.

Reply:
I'd be happy to implement a minor mode like this! Let me know if the updated solution is feasible for now, if not, I will work on this instead!